release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4429,4 +4429,29 @@ $ oc adm release info 4.12.48 --pullspecs
 [id="ocp-4-12-48-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+
+[id="ocp-4-12-49"]
+=== RHSA-2024:0664 - {product-title} 4.12.49 bug fix and security update
+
+Issued: 2024-02-09
+
+{product-title} release 4.12.49, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0664[RHSA-2024:0664] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0666[RHBA-2024:0666] advisory.
```

Comment:
Did the packages advisory get switched to an RHSA maybe? The current RHBA link is not resolving.
```suggestion
{product-title} release 4.12.49, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0664[RHSA-2024:0664] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0666[RHSA-2024:0666] advisory.
```
^ This does resolve, however to a "Moderate: OpenShift Container Platform 4.12.49 packages and security update".

It's also 404'ing from the body text of https://access.redhat.com/errata/RHSA-2024:0666.

Reply:
Thanks, updated.